src/api/apiUtils.js

```diff
@@ -39,3 +39,51 @@ export function getOrganizationId() {
 export function getOrganizationUUID() {
   return session.user?.policy?.organization?.uuid || null;
 }
+
+export const makeQueryString = (filterObj) => {
+  // log.debug('makeQueryString 1 ----->', filterObj);
+  const arr = [];
```

Comment:
None of `arr`, `whereNulls`, `whereNotNulls` or `whereIns` should be declared `const`.
They're all potentially modified after declaration.